contracts/CollateralVaultCoreV2.vy

```diff
@@ -0,0 +1,165 @@
+# @version 0.3.7
+
+
+# Interfaces
+
+from vyper.interfaces import ERC165 as IERC165
+from vyper.interfaces import ERC721 as IERC721
+
+interface IDelegationRegistry:
+    def delegateForToken(delegate: address, contract_: address, tokenId: uint256, value_: bool): nonpayable
+
+# Structs
+
+
+# Events
+
+event OwnershipTransferred:
+    ownerIndexed: indexed(address)
+    proposedOwnerIndexed: indexed(address)
+    owner: address
+    proposedOwner: address
+
+event OwnerProposed:
+    ownerIndexed: indexed(address)
+    proposedOwnerIndexed: indexed(address)
+    owner: address
+    proposedOwner: address
+
+event CollateralVaultPeripheralAddressSet:
+    currentValue: address
+    newValue: address
+
+
+# Global variables
+
+vaultName: constant(String[30]) = "ERC721"
```

Comment:
What's this used for? Is it for the delegation? Can we then give it a better name that also ties it to Zharta, like `ZhartaERC721`?

Reply:
Just to be used by `LiquidationsPeripheral.vy`. I'll change to lowercase for consistency.